src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -1,15 +1,13 @@
 package lotto.domain;
 
-import lotto.constant.Constants;
-
 import java.util.HashMap;
 
 public class LottoNumber {
-    private int number;
-    private static HashMap<Integer, LottoNumber> lottoNumbers = new HashMap<>();
+    private static final HashMap<Integer, LottoNumber> lottoNumbers = new HashMap<>();
```

Comment:
여기도 HashMap이 아니라 Map Type을 써서 하는게 좋을거 같습니다.

```suggestion
    private static final Map<Integer, LottoNumber> lottoNumbers = new HashMap<>();
```